docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,7 +1,202 @@
 ---
-title: "Getting started with Dagster Cloud | Dagster Docs"
+title: Getting started with Dagster Cloud | Dagster Cloud
 
-noindex: True
+platform_type: "cloud"
 ---
 
-# Getting started with Dagster Cloud
+# Getting Started with Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Welcome to Dagster Cloud!
+
+In this guide, we'll give you everything you need to get up and running with Dagster Cloud. We'll walk you through setting up your Dagster Cloud account, selecting a deployment type, loading your code, launching your first run, and configuring continuous integration.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need access to some Dagster code.
+
+You can either use your own code or our [sample code (`dagster-cloud-examples`)](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). This guide uses the `dagster-cloud-examples` Python package.
+
+---
+
+## Step 1: Create a Dagster Cloud account and organization
+
+If you don't already have a Dagster Cloud account, [sign up for one](https://dagster.cloud/) before continuing.
+
+You'll also be asked to create a **Dagster Cloud organization.** When your organization is created, a single instance deployment, named `prod`, will also be created. Instance deployments are similar to environments in that they allow you to operate independent instances of Dagster, with separately managed permissions.
```

Comment:
My preference here would be to use just `deployment` vs `instance deployment` for concision's sake, especially since it's already defined as an `independent instance` here - that also matches what's in the product.

The phrase `deployments are similar to environments` feels a bit tricky to me, since I'm not sure what environment means in this context. To me, this reads as if `environment` is an object or noun in the Dagster ecosystem, rather than a reference to the general idea of an environment. A tweak to something like `Deployments are standalone environments, allowing you to operate independent instances of Dagster...` is a bit more parseable to me.

Not sure if we have standardized this, but since **Dagster Cloud organization** is bolded, maybe  **deployment** should be too, to highlight that this is where it's being defined.

Reply:
I agree. I struggled to write this in a way that didn't sound weird, honestly. I like your suggestions and have implemented them in the rewrite.